src/window.rs

```diff
@@ -34,7 +38,6 @@ fn handle_new_grid_size(new_size: LogicalSize, renderer: &Renderer) {
     if new_size.width > 0 && new_size.height > 0 {
         let new_width = ((new_size.width + 1) as f32 / renderer.font_width) as u32;
         let new_height = ((new_size.height + 1) as f32 / renderer.font_height) as u32;
-        // Add 1 here to make sure resizing doesn't change the grid size on startup
         BRIDGE.queue_command(UiCommand::Resize {
             width: new_width,
```

Comment:
I believe this is still necessary. The conversion here isn't about dpi. Its for converting pixel sizes to grid sizes. So this change should be reverted or else window resize will seriously mess things up